source/reference/cutover-process.txt

```diff
@@ -63,6 +88,19 @@ Steps
       migration, you must issue a commit request for each ``mongosync``
       instance.
 
+      Request
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/commit.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/success.json
+         :language: json
+         :copyable: false
+
       .. note:: 
 
          Ensure that the ``mongosync`` process response for the
```

Comment:
I'm a little confused about this note and how it relates to the preceding response. The response is  `{"success":true}`, but the note says:

"Ensure that the mongosync process response for the commit request indicates that the mongosync state is COMMITTING or COMMITTED"

I don't see that in the example response we're adding here. Maybe I'm misunderstanding something here?

Reply:
I believe that the mongosync state is not explicitly returned in the response when users call the commit endpoint, which I agree contradicts what is here. I will clarify that in this note.